### ISSUE TITLE
[Accessibility Docs] Search and filter

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/vanilla-framework#4061]
Using [this template](https://docs.google.com/document/d/1IwFMfJrTfJPXDcVJfLWXYkw-pjgWhBWo4GX8X098Q64/edit#), create accessibility docs for [Search and filter](https://vanillframework.io/docs/patterns/search-and-filter)

**Comment by saiteja2108** [comment]
can i contribute on this

**Comment by bartaz** [comment]
@saiteja2108 Hey, thanks for the interest in the issue.

Contributing to this specific one can be quite difficult as it's a complex component.
We have quite a few other components missing accessibility docs (you can find them here: https://github.com/canonical/vanilla-framework/issues?q=is%3Aissue+is%3Aopen+%22%5Baccessibility+docs%5D%22)

Even if they are currently assigned we are not working on them, so feel free to pick any one you would feel comfortable with. Descriptions link to specific component pages. And the goal is to have this kind of accessibility documentation for them: https://vanillaframework.io/docs/patterns/accordion/accessibility

**Comment by bartaz** [comment]
Triaging: High effort (docs don't exist), impacts accessibility (documentation only).
Should be addressed in next architecture docs. Not doing as part of v4.
